testing/dhcpdump/dhcpdump-fixes.patch

```diff
@@ -0,0 +1,112 @@
+--- dhcpdump-1.8.orig/dhcpdump.c
```

Comment:
Could you write a comment at the start of the file what the patch does and why it's needed?